CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Zlux App Manager will be documented in this file.
 
+## `1.28.4`
+
+- Bugfix: Fixed Iframes from unintentionally loading their sources multiple times during refocus & multi-app situations
+- Bugfix: Replace existing snapshot preview with lighter UI to significantly increase multi-app Desktop performance
```

Comment:
Please put at the end of these (#350). Its something we want to do in general. It helps the release notes.